src/index.js

```diff
@@ -532,6 +532,12 @@ export default class RNPickerSelect extends PureComponent {
                         onValueChange={this.onValueChange}
                         selectedValue={selectedItem.value}
                         {...pickerProps}
+                        onFocus={() => {
+                            Keyboard.dismiss();
+                            if (pickerProps.onFocus) {
```

Comment:
I'm not sure about how this repo does it, but in E/App we would also check to make sure that `onFocus` is also a function before calling it. Should we do that here? It seems like it would be nice, so that the lib doesn't crash if someone passes something that isn't a function.

Reply:
That was what I initially thought too. But I check for the other callback (`onOpen`, `onClose`) and none of them check if it's a function. I think it's fine to let the app that implement the library to make sure a function is passed, just like React Native component that will throw an error if you give `onPress` an integer.

Reply:
OK, I think that works for me. I'll be interested to see if they request something similar in the upstream repo's PR.